puli.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
is this file supposed to be committed? what about other factories, can they come with their own puli file? and should the binding not be in either discovery or the adapter repo?